openfoodfoundation/openfoodnetwork
# [Invoices] Actions dropdown menu disappears after creating the first invoice

#### What? Why?

- Closes #11049





#### What should we test?
##### Test the feature
- As a shop manager, go to an order and its invoice panel 
- Click on the `New Invoice` button
<img width="1413" alt="Capture d’écran 2023-06-21 à 10 26 07" src="https://github.com/openfoodfoundation/openfoodnetwork/assets/296452/d971ff73-5d6f-4030-9b92-a8d51bdc6508">
- Check that an invoice has been generated and the dropdown `Actions` is still functional. 
<img width="1396" alt="Capture d’écran 2023-06-21 à 10 26 15" src="https://github.com/openfoodfoundation/openfoodnetwork/assets/296452/af1a6ba2-7069-4979-bc37-243b2f3a7456">


##### Check there is no regression
As a shop manager, check that, in different context, the dropdown `Actions` for an order has no regression.

#### Release notes



Changelog Category: User facing changes



The title of the pull request will be included in the release notes.
